Makefile

```diff
@@ -136,7 +136,7 @@ else
 ifeq ($(DEBUG),INFO)
 DEBUG_FLAGS            = -ggdb3
 endif
-OPTIMISATION_BASE     := -flto -fuse-linker-plugin -ffast-math -fmerge-all-constants
+OPTIMISATION_BASE     := -flto=auto -fuse-linker-plugin -ffast-math -fmerge-all-constants
```

Comment:
Can you point us at the documention on the lto=auto change?  Currently we depend on lto to optimize the whole program, we don't want a situation where a minor change causes lto to be disabled resulting in a code that no-longer fits in the target mcu.

I looked, but only found this:

https://gcc.gnu.org/legacy-ml/gcc-patches/2019-07/msg01488.html

I checked here:

https://gcc.gnu.org/onlinedocs/gccint/LTO-Overview.html

and here (see `-flto[=n]`) 

https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html

but there was no mention of `auto`.

Reply:
https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html

> Use -flto=auto to use GNU make’s job server, if available, or otherwise fall back to autodetection of the number of CPU threads present in your system.